lib/localize.js

```diff
@@ -166,6 +189,9 @@ export const getLocalizeClass = (superclass = class {}) => class LocalizeClass e
 			const possibleLanguages = this._generatePossibleLanguages(config);
 			const resourcesPromise = this.getLocalizeResources(possibleLanguages, config);
 			resourcesLoadedPromises.push(resourcesPromise);
+			resourcesLoadedPromises.push(this.getLocalizeResources(possibleLanguages, {
```

Comment:
I just realized I can maybe leverage the `localizeConfig` to signal whether or not to load the common stuff -- I'll give that a try and see how it looks.